src/TestFramework/TestFramework.Extensions/MessageLevel.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestTools.UnitTesting;
+
+/// <summary>
+/// An enumeration to specify the level of the messages to be displayed when using TestContext.DisplayMessage API.
+/// </summary>
+public enum MessageLevel
+{
+    /// <summary>
+    /// The message will be displayed in informational level.
+    /// </summary>
+    Informational,
+
+    /// <summary>
+    /// The message will be displayed in warning level.
+    /// </summary>
+    Warning,
+
+    /// <summary>
+    /// The message will be displayed in error level.
+    /// </summary>
+    Error,
+}
```

Comment:
```suggestion
/// <summary>
/// Specifies the severity level of messages displayed using the <see cref="TestContext.DisplayMessage"/> API.
/// </summary>
public enum MessageLevel
{
    /// <summary>
    /// The message will be displayed as informational, typically used for general updates or non-critical messages.
    /// </summary>
    Informational,

    /// <summary>
    /// The message will be displayed as a warning, indicating a potential issue or something requiring attention.
    /// </summary>
    Warning,

    /// <summary>
    /// The message will be displayed as an error, representing a significant issue or failure.
    /// </summary>
    Error,
}
```

Reply:
IIRC, cref for TestContext doesn't work as this project doesn't have access to it.

Reply:
I must be confusing different things. TestContext is actually accessible from here. Will update to use cref